### PR TITLE
linux: Adding IGT Chamelium test configs

### PR DESCRIPTION
**Pull request by nareshkamboju**
IGT GPU Tools is a collection of tools for development and testing of
the DRM drivers. LKFT enabling IGT test plan so we need IGT pre required
kernel config fragments.

Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>

### CODE REVIEW

**Review by mrchapp — CHANGES_REQUESTED** [review]
Please add those 2 sections (with comments) to lkft.config, so that we can reuse them with LKFT 2.0 too.

**Comment by nareshkamboju** [comment]
@mrchapp your review comments addressed.
Thanks for the review.

File: recipes-kernel/linux/files/lkft.config [hunk 1]
```diff
@@ -92,3 +92,47 @@ CONFIG_QEDE=y
...
+CONFIG_SND_SOC_QDSP6_ROUTING=m
+CONFIG_SND_SOC_QDSP6_ASM=m
+CONFIG_SND_SOC_QDSP6_ASM_DAI=m
+CONFIG_SND_SOC_QDSP6_CORE=m
+CONFIG_SND_SOC_QDSP6=m
+CONFIG_SND_SOC_MSM8996=m
+CONFIG_SND_SOC_MSM8916_WCD_ANALOG=m
+CONFIG_SND_SOC_MSM8916_WCD_DIGITAL=m
+#
+# USB support
```
Comment (roxell) [inline recipes-kernel/linux/files/lkft.config]: we have USB support above in the file...

File: recipes-kernel/linux/files/lkft.config [hunk 2]
```diff
@@ -92,3 +92,47 @@ CONFIG_QEDE=y
...
+#
+CONFIG_USB_ULPI_BUS=y
+CONFIG_GENERIC_PHY=y
+CONFIG_EXTCON=y
+CONFIG_PHY_QCOM_QMP=y
```
Comment (roxell) [inline recipes-kernel/linux/files/lkft.config]: we already enable this

File: recipes-kernel/linux/files/lkft.config [hunk 3]
```diff
@@ -92,3 +92,47 @@ CONFIG_QEDE=y
...
+CONFIG_PHY_QCOM_QUSB2=y
+CONFIG_PHY_QCOM_UFS=y
```
Comment (roxell) [inline recipes-kernel/linux/files/lkft.config]: and this

Comment (nareshkamboju) [reply recipes-kernel/linux/files/lkft.config]: @roxell  Thanks for the review.
The duplicate entries have been deleted and pull request updated.

**Review by mrchapp — APPROVED** [review]

File: recipes-kernel/linux/files/lkft.config (hunk 1 above)
Comment (nareshkamboju) [reply recipes-kernel/linux/files/lkft.config]: @roxell  deleted duplicate config fragments
and USB support configs changes committed in a separate commit.

**Pull request merged**